aspnetcore/includes/benefits.md

```diff
@@ -1,21 +1,85 @@
 ASP.NET Core provides the following benefits:
 
-* A unified story for building web UI and web APIs.
+:::moniker range=">= aspnetcore-6.0"
+
+<!-- AUTHOR NOTE: >=6.0 content showcases Blazor and 
+                  Minimal APIs, demotes RP/MVC -->
+
+* A unified story for building web apps, web APIs, Azure IoT (Internet of Things) apps, and mobile backends.
+* Architected for testability.
+* [Blazor](xref:blazor/index) lets you create rich interactive client-side UIs using [.NET](/dotnet/standard/tour) and [C#](/dotnet/csharp/) with wide browser support based on HTML/JavaScript, including mobile browsers. You can also build hybrid desktop and mobile apps with .NET and Blazor.
```

Comment:
Blazor isn't just for client-side. It does server-side rendering too.

```suggestion
* Create rich interactive web UI with [Blazor](xref:blazor/index) using [C#](/dotnet/csharp/) - no JavaScript required. Reuse your Blazor components to build native mobile and desktop apps with [Blazor Hybrid](xref:blazor/hybrid/index).
```

Reply:
The style manual will want an em dash there ... `&mdash;` ... &mdash; ... with no spaces. I'll edit your suggestion to apply it.